charts/delphai-keda/values.yaml

```diff
@@ -29,4 +29,4 @@ env: []
 gpu: false
 mountPath: /app/data
 fileShares: []
-podAntiAffinity: false
+onePodMaxPerNodePerApp: false
```

Comment:
I added `Max` to express that it's one pod per node per app maximum, so that it implies that it's no more than one pod per node for the same app